src/repos/admin.js

```diff
@@ -14,23 +14,78 @@ function isAdmin(db, id) {
   })
 }
 
-function getAdmins(db) {
+function saveAdmin(db, admin) {
+  var upsert = sql`
+  INSERT INTO admin (
+    steam_id,
+    group_id,
+    division_id
+  ) VALUES (
+    ${admin.steam_id},
+    ${admin.group_id},
+    ${admin.division_id}
+  ) ON CONFLICT (
+    steam_id
+  ) DO UPDATE SET
+    group_id = ${admin.group_id},
+    division_id = ${admin.division_id}
+  `
+  return db.query(upsert)
+}
+
+function getAdmins(db, criteria) {
   var select = sql`
   SELECT
     steam_user.steam_id,
     steam_user.avatar,
     COALESCE(profile.name, steam_user.name) AS name,
-    admin.title,
-    admin.description
+    admin.group_id,
+    admin.division_id,
```

Comment:
NCN: Generally it's better to use the "source of truth" for this sort of field - in this case `division.id`.